src/state/README.md

```diff
@@ -54,7 +54,7 @@ Used within `mapStateToProps` in containers.
 
 Each branch of the Redux state tree has its own directory under the `state` directory. That subdirectory serves as a module, bundling together action constants (`constants.ts`),
 action creators (`actions.ts`), a single reducer (`reducer.ts`), selectors (either in `selectors.ts` if there are few, or in their own subfolder, `selectors/` if there are many), associated types
-(`types.ts`), and any logics (if few, `logics.ts`, if many, under own subfolder, `logics/`). Each of these state directories has an `index.ts` to package the bundle. This pattern loosely follows the 
+(`types.ts`), and any logics (if few, `logics.ts`, if many, under own subfolder, `logics/`). Each of these state directories has an `selectors.ts` to package the bundle. This pattern loosely follows the 
```

Comment:
I think this was an overzealous Intellij auto-refactor? It should be `index.ts`....

Reply:
Ha, yeah it was